### PR TITLE
fix(datastore): Ensure the datastore time is returned as UTC

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: datastore/load.go [hunk 1]
```diff
@@ -536,7 +536,7 @@ func propToValue(v *pb.Value) (interface{}, error) {
 	case *pb.Value_DoubleValue:
 		return v.DoubleValue, nil
 	case *pb.Value_TimestampValue:
-		return time.Unix(v.TimestampValue.Seconds, int64(v.TimestampValue.Nanos)), nil
+		return time.Unix(v.TimestampValue.Seconds, int64(v.TimestampValue.Nanos)).In(time.UTC) nil
```
Comment (tbpg) [inline datastore/load.go:539]: Missing a comma here. :smile:

File: datastore/load.go [hunk 2]
```diff
@@ -369,7 +369,7 @@ func setVal(v reflect.Value, p Property) (s string) {
 			if ok {
 				s := micros / 1e6
 				ns := micros % 1e6
-				v.Set(reflect.ValueOf(time.Unix(s, ns)))
+				v.Set(reflect.ValueOf(time.Unix(s, ns).In(time.UTC)))
```
Comment (tbpg) [inline datastore/load.go:372]: Mind adding a test that ensures the times are returned in UTC?

**Review by tbpg — CHANGES_REQUESTED** [review]

File: datastore/load.go (hunk 2 above)
Comment (tritone) [reply datastore/load.go:372]: +1, might be easier said than done though :)

File: datastore/load.go (hunk 1 above)
Comment (crwilcox) [reply datastore/load.go:539]: Thanks @tbpg If not obvious, I need to author tests/review existing for this yet. I am trying to wrap another task, but also was hoping to have CI let me know how much is left here :) Should get back to this later today.

File: datastore/load.go (hunk 2 above)
Comment (crwilcox) [reply datastore/load.go:372]: test added.

File: datastore/load_test.go [hunk 3]
```diff
@@ -486,6 +486,21 @@ func TestLoadToInterface(t *testing.T) {
 			dst:  &withUntypedInterface{Field: 1e9},
 			want: &withUntypedInterface{Field: "Newly set"},
 		},
+		{
+			name: "struct with timestamp",
```
Comment (tritone) [inline datastore/load_test.go:490]: Just to confirm, does this test case fail currently without the fix?

**Review by tritone — COMMENTED** [review]
Looks good.

File: datastore/load_test.go [hunk 4]
```diff
@@ -584,6 +569,36 @@ func TestLoadToInterface(t *testing.T) {
 	}
 }
 
+func TestTimezone(t *testing.T) {
+	// Expect Local times to be represented in UTC
```
Comment (tritone) [inline datastore/load_test.go:573]: I'd put this comment above L572 since it describes the whole test.

File: datastore/load_test.go [hunk 5]
```diff
@@ -584,6 +569,36 @@ func TestLoadToInterface(t *testing.T) {
...
+		Time: time.Unix(1605504600, 0).In(time.UTC),
+	}
+
+	err := loadEntityProto(dst, src)
+
+	if err != nil {
+		t.Fatalf("loadEntityProto: %v", err)
+	}
+
+	gotZone, _ := dst.Time.Zone()
```
Comment (tritone) [inline datastore/load_test.go:594]: Maybe actually compare dst to want as well, and note why you need to compare the time zones separately.

**Review by tbpg — APPROVED** [review]
Thanks!

**Pull request merged**